packages/riverpod/lib/src/async_notifier/base.dart

```diff
@@ -440,16 +443,43 @@ mixin FutureHandlerProviderElementMixin<T>
   }
 }
 
+/// The element of [AsyncNotifierProvider].
+abstract class AsyncNotifierProviderElementBase<
+        NotifierT extends AsyncNotifierBase<T>,
+        T> extends ProviderElementBase<AsyncValue<T>>
+    with FutureHandlerProviderElementMixin<T> {
+  AsyncNotifierProviderElementBase._(super.provider);
+
+  final _notifierNotifier = ProxyElementValueNotifier<NotifierT>();
```

Comment:
_notifierNotifier is it mistake ?

Reply:
It's a notifier for the ".notifier" modifier
Kinda weird but that's how the naming convention for the other similar variables is.